examples/spire/webhook/clusterspiffeid-webhook-template.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: spire.spiffe.io/v1alpha1
+kind: ClusterSPIFFEID
+metadata:
+  name: nsm-workloads-webhook
+spec:
+  spiffeIDTemplate: "spiffe://{{.TrustDomain}}/ns/{{.PodMeta.Namespace}}/pod/{{.PodMeta.Name}}"
+  dnsNameTemplates: ["{{index .PodMeta.Labels \"dns-name\"}}.{{.PodMeta.Namespace}}", "{{index .PodMeta.Labels \"dns-name\"}}.{{.PodMeta.Namespace}}.svc"]
+  podSelector:
+    matchLabels:
+      spiffe.io/spiffe-id-webhook: "true"
```

Comment:
I think it would be better to move this file into `examples/spire/base` folder, instead of creating a new one.

Reply:
Thanks, fixed